metricflow/query/query_parser.py

```diff
@@ -6,7 +6,11 @@
 from dataclasses import dataclass
 from typing import List, Optional, Tuple, Dict, Sequence
 
-import fuzzywuzzy.fuzz
+import warnings
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=UserWarning)
+    import fuzzywuzzy.fuzz
```

Comment:
> Note: this can only be guaranteed in a single-threaded application. If two or more threads use the [catch_warnings](https://docs.python.org/3/library/warnings.html#warnings.catch_warnings) context manager at the same time, the behavior is undefined.

From https://docs.python.org/3/library/warnings.html

MetricFlow is not single-threaded, although in the CLI the query_parser is effectively used in exactly one place. I think this is still ok given what we need here, but I don't know if there are more robust alternatives available.

There's this:

https://docs.python.org/3/library/logging.html#logging.captureWarnings

Seems like maybe we could enable capture, suppress that one warning during the import, and then disable capture.

@plypaul did you have any other ideas in mind?

Reply:
> If two or more threads use the [catch_warnings](https://docs.python.org/3/library/warnings.html#warnings.catch_warnings) context manager at the same time, the behavior is undefined.

Yeah but this sounds exactly like a classic Python Logger thing to do.